src/sx-tailwind-website/src/components/Sidebar.js

```diff
@@ -56,7 +56,8 @@ export default function Sidebar({ screenSize = 'large' }: Props): Node {
           <div className={tailwind('mt-5 flex-1 flex flex-col overflow-y-auto')}>
             <div className={tailwind('overflow-y-auto')}>
               <nav className={tailwind('px-2')}>
-                <MenuItem label="Documentation" href="/" screenSize={screenSize} />
+                <MenuItem label="Intro" href="/" screenSize={screenSize} />
+                <MenuItem label="Documentation" href="/tailwind-css/" screenSize={screenSize} />
```

Comment:
The menu item doesn't stay marked as open when being on this page (in production). The rest of the pages work correctly.

Reply:
#1283